charts/dataingestion-service/values.yaml

```diff
@@ -84,10 +84,12 @@ pvc:
 
 efsFileSystemId: "EXAMPLE_EFS_ID"
 
-authUri: ""
+authUri: "http://keycloak.keycloak.svc.cluster.local/realms/NBS"
```

Comment:
Does the URI include http in this case?

Reply:
it appears to from the DI documents, thinking that may need revision at some point wrt MTLS etc